src/core/server/integration_tests/saved_objects/migrations/group3/actions/actions.test.ts

```diff
@@ -93,7 +93,7 @@ describe('migration actions', () => {
     await bulkOverwriteTransformedDocuments({
       client,
       index: 'existing_index_with_docs',
-      operations: docs.map(createBulkIndexOperationTuple),
+      operations: docs.map((doc) => createBulkIndexOperationTuple(doc)),
```

Comment:
Just curious, what issue was this causing? I can see it is safer if `createBulkIndexOperationTuple` takes more than one arg.

Reply:
`createBulkIndexOperationTuple` was changed in this PR to accept 2 arguments, and `map` pass the index as second argument, so it creates a argument type mismatch